frontend/test/metabase/scenarios/question/native.cy.spec.js

```diff
@@ -229,7 +229,6 @@ describe("scenarios > question > native", () => {
     cy.findByText("Simple question").click();
     popover().within(() => {
       cy.findByText("Saved Questions").click();
-      cy.findByText("Robert Tableton's Personal Collection").click();
```

Comment:
Everything looks good. Why was this change needed?

Reply:
That's great you asked, I was not 100% sure and went to double-check and there is actually an unexpected behavior here